### PR TITLE
Feature/ui

### PR DESCRIPTION
**Pull request by rlacorne**
Base stuff

### CODE REVIEW

File: client/src/components/Layout/index.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
-import styled from 'styled-components';
+import styled from 'styled-components'; 
 
 interface ILayoutProps {
   children: React.ReactNode;
   className?: string;
+  theme: any;
```
Comment (Allov) [inline client/src/components/Layout/index.tsx]: should be `IAppTheme`

File: client/src/lib/styled/interface.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  pBackground: nightable.background.night,
+  pText: nightable.text.night,
+}
+
+const day: IAppTheme = {
+  pBackground: nightable.background.day,
+  pText: nightable.text.day,
+}
+
+const party: IAppTheme = {
```
Comment (Allov) [inline client/src/lib/styled/interface.tsx]: should probably use `nightable` (themeable?)

File: client/src/index.tsx [hunk 3]
```diff
@@ -5,9 +5,10 @@ import App from './containers/App';
 import registerServiceWorker from './registerServiceWorker';
 import configureStore, { history } from './store';
 
+
```
Comment (Allov) [inline client/src/index.tsx]: useless whitespace

File: client/src/lib/styled/interface.tsx (hunk 2 above)
Comment (rlacorne) [reply client/src/lib/styled/interface.tsx]: Pas vraiment, party cest un theme in itself, qui a pas de verion night.

A moins qu' on aie partyDay et partyNight ?

File: client/src/components/Layout/index.tsx (hunk 1 above)
Comment (rlacorne) [reply client/src/components/Layout/index.tsx]: IAppTheme: any, ou theme: IAppTheme ?

Comment (Allov) [reply client/src/components/Layout/index.tsx]: `theme: IAppTheme`

Comment (rlacorne) [reply client/src/components/Layout/index.tsx]: tep, push updated

**Pull request merged**